tests/test_plugin.py

```diff
@@ -231,3 +251,52 @@ def test_image_product(dispatcher_live_fixture, mock_backend):
     assert c.status_code == 200
     imdata = jdata['products']['numpy_data_product_list'][0]
     oda_ndp = ImageDataProduct.decode(imdata)
+
+def test_default_kg(conf_file, dispatcher_live_fixture):  
+    server = dispatcher_live_fixture
+    logger.info("constructed server: %s", server)
+
+    c = requests.get(server + "/instr-list",
+                    params = {'instrument': 'mock', 
+                              'token': encoded_token})
+    logger.info("content: %s", c.text)
+    jdata = c.json()
+    logger.info(json.dumps(jdata, indent=4, sort_keys=True))
+    logger.info(jdata)
+    assert c.status_code == 200 
+    assert 'lightcurve-example' in jdata # TODO: change to what will be used in docs
+
+@pytest.mark.parametrize("privileged", [True, False])
+def test_local_kg(conf_file, dispatcher_live_fixture, privileged):  
```

Comment:
where do you define `conf_file` ?

Reply:
It's here https://github.com/oda-hub/dispatcher-plugin-nb2workflow/blob/0f66d8ccd6e3a106186d10bc7524f88a109efb43/tests/conftest.py#L44